cdc-accelerator/core/sitesCache.js

```diff
@@ -40,4 +38,27 @@ export default class SitesCache {
     static getSiteInfo(_apiKey) {
         return SitesCache.cache.find(({ apiKey }) => apiKey === _apiKey)
     }
+
+    static isCacheUpdated(configuration) {
+        if (!configuration?.cache) {
+            return false
+        }
+        let updated = true
+        if (configuration.source) {
+            configuration.source.forEach((entry) => {
+                updated &= configuration.cache.some(({ apiKey }) => {
```

Comment:
Using `&=` you are converting `updated` to a number, it's a good practice to not change the type of the variables. You can try to use `&&=` instead.

Reply:
Good catch :) changed